fiftyone/core/annotation/generate_label_schema.py

```
@@ -0,0 +1,563 @@
+                )
+
+            elif len(values) <= foac.VALUES_THRESHOLD:
+                settings[foac.COMPONENT] = foac.DROPDOWN
+
+            if settings[foac.COMPONENT] in foac.VALUES_COMPONENTS:
+                settings[foac.VALUES] = values
+
+            if settings[foac.COMPONENT] == foac.RADIO:
+                settings[foac.DEFAULT] = values[0]
+
+    except:
```

Ruff (0.14.7)

Do not use bare `except`

(E722)